bci_tester/data.py

```diff
@@ -174,6 +176,11 @@
             "registry.suse.de/suse/sle-15-sp6/update/products/ai/totest"
         )
 
+    if OS_VERSION == "15.6-pr":
+        ibs_cr_project = (
+            "registry.suse.de/suse/sle-15-sp6/update/products/privateregistry"
```

Comment:
Another project that cannot be tested on github?

Reply:
it can be tested on github via ibs-released mode, which we can enable once we actually managed to release it.

Reply:
We'd have to ensure that this is doesn't default to ibs-released on openQA, but otherwise I'd be very much in favor